parser/src/tokenizer/token.rs

```diff
@@ -18,35 +18,20 @@ use serde::{Deserialize, Serialize};
 use std::fmt;
 use tendril::StrTendril;
 
-/// Parts of a formatted string for logging to the console.
-#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
-pub enum FormatStringPart {
-    Const(#[serde(with = "leo_ast::common::tendril_json")] StrTendril),
-    Container,
-}
-
-impl fmt::Display for FormatStringPart {
-    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        match self {
-            FormatStringPart::Const(c) => write!(f, "{}", c),
-            FormatStringPart::Container => write!(f, "{{}}"),
-        }
-    }
-}
-
 /// Represents all valid Leo syntax tokens.
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
 pub enum Token {
     // Lexical Grammar
     // Literals
     CommentLine(#[serde(with = "leo_ast::common::tendril_json")] StrTendril),
     CommentBlock(#[serde(with = "leo_ast::common::tendril_json")] StrTendril),
-    FormatString(Vec<FormatStringPart>),
+    StringLit(Vec<char>),
```

Comment:
can we use string here too?

Reply:
At which stage then we should parse characters? Current implementation is to parse each character in parser and pass parsed (by character) string.